src/test/java/com/epam/tc/nitcenkov/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,101 @@
+package com.epam.tc.nitcenkov.hw2.ex1;
+
+import com.epam.tc.nitcenkov.hw2.GeneralExerciseTest;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class FirstExerciseTest extends GeneralExerciseTest {
+
+    @Test
+    public void firstExerciseTest() {
+
+        //Open test site by URL
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //Assert Browser title
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+
+        //Perform login
+
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(
+            webDriver.findElement(By.id("user-icon")))).click();
+
+        webDriver.switchTo().activeElement();
+
+        webDriverWait.until(ExpectedConditions
+            .visibilityOf(webDriver.findElement(By.id("name")))).sendKeys("Roman");
+
+        webDriverWait.until(ExpectedConditions
+            .visibilityOf(webDriver.findElement(By.id("password")))).sendKeys("Jdi1234");
+
+        webDriverWait.until(ExpectedConditions
+            .visibilityOf(webDriver.findElement(By.id("login-button")))).click();
+
+        //Assert Username is logged
+        String actualUserName =
+            webDriverWait.until(ExpectedConditions.visibilityOf(webDriver.findElement(By.id("user-name")))).getText();
+        softAssertions.assertThat(actualUserName).isEqualTo("ROMAN IOVLEV");
+
+        //Assert that there are 4 items on the header section are displayed and they have proper texts
+        webDriverWait.until(ExpectedConditions
+            .visibilityOfAllElements(
+                webDriver.findElements(By.xpath("//*[@class='uui-navigation nav navbar-nav m-l8']"))));
+        List<WebElement> menuButtons =
+            webDriver.findElements(By.xpath("//*[@class='uui-navigation nav navbar-nav m-l8'][1]/li"));
```

Comment:
Locator type precedence violation: DOM, CSS, XPATH
Please combine these two statements

\+ same for following collections